src/db/model/user.rs

```diff
@@ -8,6 +8,7 @@ use serde_json;
 pub struct User {
   pub id: u64,
   pub role: String,
+  pub status: String,
```

Comment:
I think this should be some sort of enum, if Diesel allows that

Reply:
Diesel does not (currently) support enum types for mysql

Reply:
On that note... can we do something about it in Rust? maybe a mapping or a special String type.. if not, at least a comment describing the values works

Reply:
Yes, it can be done. I have added this trello card for this: https://trello.com/c/Gxt9CIoj